src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -330,7 +341,32 @@ def _get_pool_status(
                 });
 
         return (ret, root_map, pool_root)
+    
+    def _update_progress_events(self):
+        osdmap = self.get_osdmap()
+        pools = osdmap.get_pools()
+        for pool_id in list(self._event):
+            ev = self._event[pool_id]
+            pool_data = pools[int(pool_id)]
```

Comment:
```
2019-08-21T21:03:28.521+0000 7f9b3b5fe700 -1 Traceback (most recent call last):
  File "/usr/share/ceph/mgr/pg_autoscaler/module.py", line 175, in serve
    self._update_progress_events()
  File "/usr/share/ceph/mgr/pg_autoscaler/module.py", line 350, in _update_progress_events
    pool_data = pools[int(pool_id)]
KeyError: (1,)
```

i don't really understand how come `int` could return a tuple..

see http://pulpito.ceph.com/kchai-2019-08-21_12:58:47-rados-wip-kefu-testing-2019-08-21-1445-distro-basic-mira/4236725/

Reply:
see also https://tracker.ceph.com/issues/41386

Reply:
```
2019-08-29T07:52:47.255+0000 7f691c344700 -1 Traceback (most recent call last):
  File "/usr/share/ceph/mgr/pg_autoscaler/module.py", line 175, in serve
    self._update_progress_events()
  File "/usr/share/ceph/mgr/pg_autoscaler/module.py", line 353, in _update_progress_events
    pool_data = pools[int(pool_id)]
KeyError: (1,)
```

i still ran into this issue even my branch contains the fix for https://tracker.ceph.com/issues/41386